app/lib/AssetUploader.js

```diff
@@ -104,11 +104,14 @@ class AssetUploader {
       method: 'post',
       body: formData
     }).then((response) => {
-      if (response.ok) {
-        this._finalizeFileUpload(upload, file);
-      } else {
-        this.options.onError(new AssetUploaderError("There was an error uploading the file. Please try again."));
+      if (!response.ok) {
+        throw null;
       }
+      this._finalizeFileUpload(upload, file);
+    }).catch(() => {
+      this.options.onError(
+        new AssetUploaderError("There was an error uploading the file. Please try again.")
```

Comment:
I wonder if we should include the original exception in the `AssetUploaderError` thing so clients can inspect it? (doesn't matter really, just a side thought)

Reply:
As like, a property or something?

Reply:
Yeah - it may not be possible (and it was just a random idea, happy for you to ignore)